estate/models/estate_property_offer.py

```diff
@@ -0,0 +1,70 @@
+from odoo import fields, models ,api
+from datetime import datetime, timedelta 
+from odoo.exceptions import UserError,ValidationError
+
+class EstatePropertyOffer(models.Model):
+    _name = "estate.property.offer"
+    _description = "Estate Property Offers"
+    
+    
+    price = fields.Float()
+    status = fields.Selection(
+        selection=[
+            ('Accepted', 'Accepted'),
+            ('Refused', 'Refused'),
+        ],
+        string="Status"
+    )
+    partner_id = fields.Many2one('res.partner', required=True)
+    property_id = fields.Many2one('estate.property', required=True)
+    validity = fields.Integer(default=7)
+    deadline_date = fields.Date(compute="_compute_validity", inverse="_inverse_validity",store=True)
+    created_date = fields.Date(default=lambda self: datetime.today())
```

Comment:
```suggestion
    create_date = fields.Date(default=lambda self: datetime.today())
```
I think we can override the existing create_date field here instead a new field.